docs/bundle/multiple_queues.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: default
+parent: "Symfony bundle"
+title: Functional testing
+nav_order: 13
+---
+{% include support.md %}
+
+# Multiple queues
+
+One of the most powerful features of this bundle is the ability to run multiple queues with the same transport/client. 
+
+## Different queues for topic routing and processing
+
+In the default configuration both the topic routing and the job processing tasks are written to the `default` queue. If you want to separate those two types of tasks you'll have to change the [configuration](config_reference.md) in `config/packages/enqueue.yaml` like so:
+
+```yaml
+enqueue:
+    default:
+        transport:            # Required
+        client:
+            traceable_producer:   true
+            prefix:               enqueue
+            separator:            .
+            app_name:             app
+            router_topic:         default
+            router_queue:         router # default is: default
+            router_processor:     null
+            redelivered_delay_time: 0
+            default_queue:        default
+```
+
+## Multiple topic queues
+
+Configuration for the different queues happens on the [processor](message_processor.md)/subscriber level - the queue is either defined in `config/services.yml` like so:
+
+```yaml
+# src/AppBundle/Resources/services.yml
+
+services:
+  AppBundle\Async\SayHelloProcessor:
+    tags:
+        # registers as topic processor
+        - { name: 'enqueue.processor', topic: 'hello', queue: 'say' }
+```
+
+or in the processor itself like so:
+
+```php
+<?php
+use Enqueue\Client\CommandSubscriberInterface;
+use Interop\Queue\Processor;
+
+class SayHelloProcessor implements Processor, TopicSubscriberInterface
+{
+    public static function getSubscribedTopics()
+    {
+        return [
+           'topic' => 'hello',
+           'queue' => 'say',
+       ];
+    }
+}
+```
```

Comment:
I might be wrong here, but doesn't `TopicSubscriberInterface` work because it creates the above `tag` during autoconfiguration of Symfony `Container`? There was also a question regarding multiple `clients` (https://github.com/php-enqueue/enqueue-dev/issues/835) which might be mentioned here (specifically - disabling default tag and attaching to a specific client)

Reply:
You're not wrong - I just figured that the client use case already has an example in `message_processor.md`